src/graphql/resolvers/queries/workshop.js

```diff
@@ -17,5 +19,13 @@ export const fieldResolvers = {
       if (!parent.tags) return [];
       return parent.tags;
     },
+    favoritedBy: async ({ id }, __, { dataSources: { firestore } }) =>
```

Comment:
you can drop the async. you end up wrapping a promise in a promise since you're just returning a promise.